packages/backend/src/registration/functions.ts

```diff
@@ -194,6 +200,14 @@ export const sendCertificateRegistrationRequest = async (
 
     const parsedCsr = await loadCSR(userData.csr)
     const username = getReqFieldValue(parsedCsr, CertFieldsTypes.nickName)
+    if (!username) {
+      log.error(`Could not parse certificate for field type ${CertFieldsTypes.nickName}`)
+      return {
+        // Should be internal server error code 500
```

Comment:
Maybe create a task for it, to not forget about it

Reply:
https://github.com/TryQuiet/quiet/issues/1514